content/02.main-text.md

```diff
@@ -151,6 +151,18 @@ For example, figures can be specified using versioned URLs that refer to the cod
 In addition, manuscripts can be templated, so that numerical values or tables get inserted directly from the repository that created them.
 An [example repository](https://github.com/greenelab/manubot-rootstock) demonstrates Manubot's features and serves as a template for users to write their own manuscript with Manubot.
 
+Since its creation to facilitate the Deep Review, Manubot has been used to write a variety of scholarly documents.
+The [Sci-Hub Coverage Study](https://github.com/greenelab/scihub-manuscript) --- performed openly on GitHub from its inception --- investigated Sci-Hub's repository of pirated articles [@doi:10.7554/eLife.32822].
+Sci-Hub [reviewed](https://github.com/greenelab/scihub-manuscript/issues/17) the initial preprint from this study in a series of tweets, pointing out a major error in one of the analyses.
+Within hours, the authors used markdown's strikethrough formatting in Manubot to cross-out the errant sentences ([commit](https://github.com/greenelab/scihub-manuscript/commit/8fcd0cd665f6fb5f39bed7e26b940aa27d4770ba), [versioned manuscript](https://greenelab.github.io/scihub-manuscript/v/8fcd0cd665f6fb5f39bed7e26b940aa27d4770ba/)), thereby alerting readers to the mistake and preventing further propagation of misinformation.
```

Comment:
I think capital "M" for Markdown (at least that's how Wikipedia stylizes it).

Reply:
Currently, we're inconsistent in our capitalization of markdown. We use lowercase more than uppercase. However, it looks like Markdown is perhaps more standard, as evidenced by Wikipedia and http://commonmark.org.

I'd suggest a future pull request changes all markdown instances to Markdown. I'll change it in this PR.

Reply:
Good idea. I'll do so tomorrow.